.cases/kafka-create.md

```diff
@@ -0,0 +1,19 @@
+## Kafka provider verification
+
+## Requirements
+
+ - Create new account 
+
+## Cases
+
+rhoas command
+> output
+
+rhoas kafka create --name=wtrocki --provider=azure
+❌ azure is not a valid or currently enabled cloud provider name. Valid names are: aws 
+
+rhoas kafka create --name=wtrocki --provider=aws --region="magentadrive"
+❌ Magentadrive is not a valid or enabled region name.
```

Comment:
Actually the error message accuracy could be problematic as we want the ID to not be transformed in any way. The error message first letter is capitalised automatically. The user may think that the CLI is inputting the capital letter during the creation which causes the invalid error.

![Screenshot 2022-01-17 at 10 56 07](https://user-images.githubusercontent.com/11743717/149757348-de1aacca-0c17-45b0-8530-dd1b180bb61b.png)

"Ibm" should remain "ibm". What we need to do is make proper use of Error types. What we could do is create generic base errors and wrap them. So there could be a `NeverCapitaliedFirstLetterBaseError` which is used when you never want the first letter to be capitalized. When the error is unwrapped you type check and only capitalize if it is not an error of this type. It might not be the best way to do it but it is possible.

Reply:
Or change message format.